Sources/GryphonLib/SharedUtilities.swift

```diff
@@ -468,3 +468,22 @@ extension Utilities {
 		return nil
     }
 }
+
+extension String {
+	func isArrayDeclaration() -> Bool {
+		if self.starts(with: "[") {
```

Comment:
I wonder why the bootstrap tests are passing for you, because they aren't passing for me... when I run `./runTests.sh -b` I get this compilation error:

````
Gryphon/Sources/GryphonLib/SharedUtilities.swift:474:11: error: unresolved reference: starts
    if (this.starts(with = "[")) {
             ^
````

That's because Gryphon doesn't know how to translate `String.starts(with: "[")`. I think you'll have to either replace this with `self.hasPrefix("[")` or teach Gryphon to translate it in [AuxiliaryFileContents.swift](https://github.com/vinivendra/Gryphon/blob/3601d30204875796f04aa55a7d9263d8bf00f0dc/Sources/GryphonLib/AuxiliaryFileContents.swift#L1476).

Reply:
I think I missed the -b flag while running the command, let me try it once again and do the required fixes.

Reply:
I have the required changes.